src/ghcup/yaml.rs

```diff
@@ -57,13 +58,27 @@ impl SnapshotStorage<SnapshotMeta> for GhcupYaml {
                 ),
             );
         });
+        eprintln!("{:#?}", snapmeta_to_remote);
 
         progress.finish_with_message("done");
 
         Ok(yaml_objs
             .into_iter()
             .map(|obj| format!("ghcup/data/{}", obj.name()))
-            .map(SnapshotMeta::force)
+            .map(|key| SnapshotMeta {
+                key,
+                last_modified: Some(
```

Comment:
Why not `last_modified: None`? Since it is `force`, no matter what `last_modified` is, it will always be re-downloaded.

Reply:
https://github.com/sjtug/mirror-clone/blob/496aeabdf1502d79d10489bdb87adf62145dc03f/src/stream_pipe.rs#L149

Here, if we have `last_modified: None`, an error will occur below.

https://github.com/sjtug/mirror-clone/blob/496aeabdf1502d79d10489bdb87adf62145dc03f/src/stream_pipe.rs#L165